<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gebrauchtwaren, Zubeh√∂r</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="main_frame">
      <property name="styleSheet">
       <string notr="true">QFrame#main_frame{
	background-color: rgb(44, 62, 80);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="header_frame">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item alignment="Qt::AlignLeft|Qt::AlignTop">
           <widget class="QFrame" name="frame">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 191, 165);
border-radius: 5px;
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="home_Button">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/Icons/icons/home.svg</normaloff>:/Icons/icons/home.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>45</width>
                 <height>45</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="web_label">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <property name="text">
                <string>X-TRAKTOR</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item alignment="Qt::AlignRight|Qt::AlignTop">
           <widget class="QFrame" name="frame_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 191, 165);
border-radius: 5px;
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="budget_label">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="inputMethodHints">
                <set>Qt::ImhNone</set>
               </property>
               <property name="text">
                <string>Budget</string>
               </property>
               <property name="textFormat">
                <enum>Qt::AutoText</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="acc_Button">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/Icons/icons/self.svg</normaloff>:/Icons/icons/self.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>45</width>
                 <height>45</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="shopping_Button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/Icons/icons/shopping-cart.svg</normaloff>:/Icons/icons/shopping-cart.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>45</width>
                 <height>45</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="content_frame">
         <property name="accessibleName">
          <string/>
         </property>
         <property name="accessibleDescription">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame#content_frame{
	background-color: rgb(44, 62, 80);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,3">
          <item>
           <widget class="QFrame" name="frame_1">
            <property name="styleSheet">
             <string notr="true">QFrame#frame_1{
	background-color: rgb(52, 73, 94);
	border-radius: 10px;
	border: 2px solid rgb(46, 204, 113);
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="name_label">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color : white;</string>
               </property>
               <property name="text">
                <string>Gebrauchtware XY</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
              <widget class="QLabel" name="picture">
               <property name="maximumSize">
                <size>
                 <width>400</width>
                 <height>400</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>:/Traktoren/pictures/Traktoren/Claas_Axion_950.jpg</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="styleSheet">
             <string notr="true">QFrame#frame_3{
	background-color: rgb(52, 73, 94);
	border-radius: 10px;
	border: 2px solid rgb(46, 204, 113);
}

QFrame#frame_3 &gt; QFrame &gt; QFrame{
	background-color: rgb(52, 73, 94);
	border-radius: 10px;
	border: 2px solid rgb(46, 204, 113);
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QFrame" name="frame_8">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QFrame" name="frame_4">
                  <property name="styleSheet">
                   <string notr="true">QFrame#frame_4{
	background-color: rgb(52, 73, 94);
	border-radius: 10px;
	border: 2px solid rgb(46, 204, 113);
}</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_10">
                   <item>
                    <widget class="QLabel" name="anz_label">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: white;</string>
                     </property>
                     <property name="text">
                      <string>Zu Verkaufen:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="anz_status">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: white;</string>
                     </property>
                     <property name="text">
                      <string>%%%</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_6">
                  <property name="styleSheet">
                   <string notr="true">QFrame#frame_6{
	background-color: rgb(52, 73, 94);
	border-radius: 10px;
	border: 2px solid rgb(46, 204, 113);
}</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <item>
                    <widget class="QLabel" name="alt_preis_label">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: white;</string>
                     </property>
                     <property name="text">
                      <string>Originalpreis:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="alt_preis_status">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: white;</string>
                     </property>
                     <property name="text">
                      <string>%%%</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_10">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_11">
                   <item>
                    <widget class="QLabel" name="kauf_label">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: white;
</string>
                     </property>
                     <property name="text">
                      <string>Kaufzeitpunkt:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="kauf_status">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: white;</string>
                     </property>
                     <property name="text">
                      <string>%%%</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_7">
                  <property name="styleSheet">
                   <string notr="true">QFrame#frame_7{
	background-color: rgb(52, 73, 94);
	border-radius: 10px;
	border: 2px solid rgb(46, 204, 113);
}</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_15">
                   <item>
                    <widget class="QLabel" name="zeit_label">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: white;</string>
                     </property>
                     <property name="text">
                      <string>Alter:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="zeit_status">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: white;</string>
                     </property>
                     <property name="text">
                      <string>%%%</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_9">
                  <property name="styleSheet">
                   <string notr="true">QFrame#frame_9{
	background-color: rgb(52, 73, 94);
	border-radius: 10px;
	border: 2px solid rgb(46, 204, 113);
}</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <widget class="QLabel" name="wert_label">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: white;</string>
                     </property>
                     <property name="text">
                      <string>Wertminderung:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="wert_status">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: white;</string>
                     </property>
                     <property name="text">
                      <string>0,00 ‚Ç¨</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_5">
                  <property name="styleSheet">
                   <string notr="true">QFrame#frame_5{
	background-color: rgb(52, 73, 94);
	border-radius: 10px;
	border: 2px solid rgb(46, 204, 113);
}</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_14">
                   <item>
                    <widget class="QLabel" name="neu_preis_label">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: white;</string>
                     </property>
                     <property name="text">
                      <string>Gesamtpreis (nach Wertverlust)</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="neu_preis_status">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: white;</string>
                     </property>
                     <property name="text">
                      <string>%%%</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_13">
         <property name="styleSheet">
          <string notr="true">QFrame#frame_13{
	background-color: rgb(52, 73, 94);
	border-radius: 10px;
	border: 2px solid rgb(46, 204, 113);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;</string>
            </property>
            <property name="text">
             <string>Kompatible Hersteller:</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="comp_label">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;</string>
            </property>
            <property name="text">
             <string>%%%</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="scroll_frame">
         <property name="styleSheet">
          <string notr="true">QFrame#scroll_frame{
	background-color: rgb(52, 73, 94);
	border-radius: 10px;
	border: 2px solid rgb(46, 204, 113);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QScrollArea" name="dyn_scrollarea">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(52, 73, 94);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContents</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>942</width>
               <height>466</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_9"/>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources/ressources.qrc"/>
 </resources>
 <connections/>
</ui>
